luatest/server.lua

```diff
@@ -127,8 +135,115 @@ function Server:new(object, extra)
     return object
 end
 
+-- Determine advertise URI for given instance from a cluster
+-- configuration.
+local function find_advertise_uri(config, instance_name, dir)
+    local urilib = require('uri')
```

Comment:
Maybe just rename `uri` to `urilib` at the module header?

Reply:
Fixed.